mappings/net/minecraft/entity/boss/dragon/EnderDragonEntity.mapping

```diff
@@ -25,10 +26,12 @@ CLASS net/minecraft/class_1510 net/minecraft/entity/boss/dragon/EnderDragonEntit
 		COMMENT (yaw, y, ?)
 	FIELD field_7027 slowedDownByBlock Z
 	FIELD field_7028 phaseManager Lnet/minecraft/class_1526;
+	FIELD field_7029 accumulatedChangeInHealth I
```

Comment:
How is accumulated important here? Otherwise this probably can be `healthChange`

Reply:
Its weird because its a field and it accumulates each time the dragon is damaged
why? i don't know

Reply:
Hmm, does it have any other usages? Or is it just an unfortunate field that's updated but never really read

Reply:
![image](https://user-images.githubusercontent.com/13246889/111050673-e7e0c300-8402-11eb-9a62-efc71292e145.png)
This is how it is used. no other place. maybe seeing the context will help a bit

Reply:
This is like damage taken while sitting.
```suggestion
	FIELD field_7029 damageDuringSitting I
```